network-api/networkapi/buyersguide/templates/fragments/product_privacy.html

```diff
@@ -4,7 +4,7 @@
 
   {% if links|length > 0 %}
   <ul class="list-unstyled mb-0">
-    {% for link in product.privacy_policy_links reversed %}
+    {% for link in links %}
```

Comment:
This is passing in all the links (if any). It's no longer being `reversed` because it's an Orderable and Orderables come with ordering in the admin.

Reply:
nice!